<<<<<<< HEAD
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserdataWindow</class>
 <widget class="QDialog" name="UserdataWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>50</y>
     <width>81</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="changePassButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeNameButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改用户名</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>311</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>311</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
font: 12pt &quot;微软雅黑&quot;;
background-color: rgb(255, 185,0);
border-radius:15px;
</string>
   </property>
   <property name="text">
    <string>  用户提出的问题</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>180</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
font: 12pt &quot;微软雅黑&quot;;
background-color: rgb(255, 185,0);
border-radius:15px;
</string>
   </property>
   <property name="text">
    <string>  用户回答的答案</string>
   </property>
  </widget>
  <widget class="QPushButton" name="messageButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>400</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
background-color: rgb(0, 185, 227);
border-radius:15px;
width:130px;
height:70px;</string>
   </property>
   <property name="text">
    <string>私信</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>113</y>
     <width>39</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
font:10pt;</string>
   </property>
   <property name="text">
    <string>获得赞</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_prs">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>113</y>
     <width>63</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
font:10pt;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>481</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255)</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_id">
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255)</string>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string>用户名</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_name">
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255)</string>
      </property>
      <property name="text">
       <string>回答数</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_ans">
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255);
font:10pt;</string>
      </property>
      <property name="text">
       <string>提问数</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_ask">
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255);
font:10pt;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_10">
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255);
font:10pt;</string>
      </property>
      <property name="text">
       <string>关注数</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_focus">
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255);
font:10pt;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_12">
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255);
font:10pt;</string>
      </property>
      <property name="text">
       <string>粉丝数</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_follow">
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255);
font:10pt;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UserdataWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UserdataWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
=======
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserdataWindow</class>
 <widget class="QDialog" name="UserdataWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>50</y>
     <width>81</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="changePassButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeNameButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改用户名</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>311</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>311</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
font: 12pt &quot;微软雅黑&quot;;
background-color: rgb(255, 185,0);
border-radius:15px;
</string>
   </property>
   <property name="text">
    <string>  用户提出的问题</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>180</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
font: 12pt &quot;微软雅黑&quot;;
background-color: rgb(255, 185,0);
border-radius:15px;
</string>
   </property>
   <property name="text">
    <string>  用户回答的答案</string>
   </property>
  </widget>
  <widget class="QPushButton" name="messageButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>400</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
background-color: rgb(0, 185, 227);
border-radius:15px;
width:130px;
height:70px;</string>
   </property>
   <property name="text">
    <string>私信</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>113</y>
     <width>39</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
font:10pt;</string>
   </property>
   <property name="text">
    <string>获得赞</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_prs">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>113</y>
     <width>63</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
font:10pt;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>481</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255)</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_id">
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255)</string>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string>用户名</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_name">
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255)</string>
      </property>
      <property name="text">
       <string>回答数</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_ans">
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255);
font:10pt;</string>
      </property>
      <property name="text">
       <string>提问数</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_ask">
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255);
font:10pt;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_10">
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255);
font:10pt;</string>
      </property>
      <property name="text">
       <string>关注数</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_focus">
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255);
font:10pt;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_12">
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255);
font:10pt;</string>
      </property>
      <property name="text">
       <string>粉丝数</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_follow">
      <property name="styleSheet">
       <string notr="true">color:rgb(255,255,255);
font:10pt;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UserdataWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UserdataWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
>>>>>>> cfdf638c3bdfa8efdbbb0c911ad54fe243c63989
